### DISCUSSION TITLE
Can Livolo TI0001 be used with other devices?

### DISCUSSION THREAD

**Discussion by wb441** [Koenkk/zigbee2mqtt#7328] [Devices]
Livolo TI0001 has the following remarks: https://www.zigbee2mqtt.io/devices/TI0001.html#important 
Basically, they can only be used with the coordinator set to:

-  channel 26 
- ext_pan_id set to [33,117,141,25,0,75,18,0]

My question: **can** it coexist with other devices? The article mentions 'you may have to setup a separate network for Livolo'. Is this always the case, or is it dependent on the other devices? Not totally clear on the technical background.

- can other devices take on the ext_pan_id forced by Livolo?
- can other devices work on ch26?

Thanks for any clarification on this.

(background: they make some very promising switches, but they're a bit expensive: https://www.ixline.eu/collections/zigbee-touch-switch/4-gang )

Grtz

**Comment by avlisiur78** [comment]
Hi, yes, in my network I have Livolo, Xiaomi and Tuya devices, and they all live on the same "space".

I believe the comment is made for someone that already have a network in place, in this cases, have to make another network, or reconfigure the existing one and all the devices...
